domains/lev1s.json

```
@@ -4,6 +4,21 @@
         "email": "lev1s@duck.com"
     },
     "records": {
-        "CNAME": "leev1s.github.io"
+        "CNAME": "leev1s.github.io",
+        "TXT": "zoho-verification=zb92672806.zmverify.zoho.com",
+        "MX": [
+            {
+                "target": "mx.zoho.com",
+                "priority": 10
+            },
+            {
+                "target": "mx2.zoho.com",
+                "priority": 20
+            },
+            {
+                "target": "mx3.zoho.com",
+                "priority": 50
+            }
+        ]
     }
```

you can't combine CNAME with other records unless it was proxied
```suggestion
"records": {
        "CNAME": "leev1s.github.io",
        "TXT": "zoho-verification=zb92672806.zmverify.zoho.com",
        "MX": [
            {
                "target": "mx.zoho.com",
                "priority": 10
            },
            {
                "target": "mx2.zoho.com",
                "priority": 20
            },
            {
                "target": "mx3.zoho.com",
                "priority": 50
            }
        ]
},
"proxied": true
```